src/sentry/api/endpoints/organization_processingissues.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.helpers.processing_issues import get_processing_issues
+from sentry.api.serializers import serialize
+from sentry.models import Project
+
+
+class OrganizationProcessingIssuesEndpoint(OrganizationEndpoint):
+    def get(self, request, organization):
+        """
+        For each Project in an Organization, list its processing issues. Can
+        be passed `project` to filter down to specific projects.
+
+        :pparam string organization_slug: the slug of the organization.
+        :qparam array[string] project: An optional list of project ids to filter
+        to within the organization
+        :auth: required
+
+        """
+        projects = Project.objects.filter(id__in=self.get_project_ids(request, organization))
```

Comment:
depending on which order these go in, one of us will need to update this to use https://github.com/getsentry/sentry/pull/11518

Reply:
Just approved, you land first and I'll update.